recipes/pyjnius/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: pyjnius
+
+  {% set tagged_version = GIT_DESCRIBE_TAG %}
+  # If current revision is not tagged add number of commits since last tag to version.
+  {% if GIT_DESCRIBE_NUMBER|int != 0 %}
+    {% set tagged_version = tagged_version + '.post' + GIT_DESCRIBE_NUMBER %}
+  {% endif %}
+
+  version: {{tagged_version}}
+
+source:
+  git_url: https://github.com/kivy/pyjnius.git
+  git_rev: "61149f20436e560cdff89e4abdd9035f8e4f7d35"
+
+build:
+  number: 0
+  string: py{{py}}_{{PKG_BUILDNUM}}
+
+requirements:
+ build:
+   - {{ compiler('c') }}
+   - {{ compiler('cxx') }}
+ host:
+   - openjdk >=8
+   - python {{PY_VER}}*
+   - cython
+   - six >=1.7.0
+   - nose
+   - ant
+ run:
+   - openjdk >=8
+   - python {{PY_VER}}*
+   - six >=1.7.0
+
+about:
+  home: https://github.com/kivy/pyjnius
+  license: MIT
+  license_file: LICENSE 
```

Comment:
Also needs the maintainers section with your GitHub handle. Snippet from the example below.

https://github.com/conda-forge/staged-recipes/blob/a6f81a02883ba59addf0fc2f15052a5d8058296e/recipes/example/meta.yaml#L75-L80